### ISSUE TITLE
Does the model have been pre-trained for different tasks（product、Reactant...）?

### ISSUE THREAD

**Issue by MJ-Zeng** [HelloJocelynLu/t5chem#10]
hello , I'm confused that whether the subtasks in the paper should be pre-trained？
Whether you need to introduce pubchem data and pre-train？

**Comment by HelloJocelynLu** [comment]
Hi, 

All T5Chem tasks can be trained directly from scratch. For the best performance, we **pretrained** the model on pubchem molecules and have the trained weights available [here](https://yzhang.hpc.nyu.edu/T5Chem/models/simple_pretrain.tar.bz2) (Yes! So the user does not need to worry about pretraining by their own, instead, they can directly download a **pretrained** model). 

Our experiments show that **finetuning** model based on this pertrained weights generally gives better performance. Another accessible [model](https://yzhang.hpc.nyu.edu/T5Chem/models/USPTO_MT_model.tar.bz2) `USPTO_500_MT` is a **finetuned** model for multi-tasking on USPTO dataset, and is ready for use without further training.

Hope it helps!

**Comment by MJ-Zeng** [comment]
Thanks,I got it.

I have a question about whether the performance of the model is improved by the introduction of pubchem data or by multi task training？

If pretrian is not used, will yield projections exceed yield-bert?（buchwald C-N coupling）

**Comment by HelloJocelynLu** [comment]
I think pretraining improves more on smaller datasets, while multi-tasking may work better for larger datasets (for example, for USPTO_MT, pretraining does not show much improvements for full dataset, but shows significant improvements on sample dataset -- but will need more experiments to support this statement. I guess it also varies case by case tho, I will definitely try pretrained model first.)
Without pretraining, yield prediction shows similar performance as yield-BERT (if we take std and experimental error into consideration)

**Comment by MJ-Zeng** [comment]
ok, thanks

**Comment by MJ-Zeng** [comment]
Excuse me again, 
the dataset USPTO_500_MT, the results of multi-task training in the test do not seem to perform better than single-task performance?
I'm not quite sure about this result.

![image](https://user-images.githubusercontent.com/38970840/191450539-660241bd-5d79-4ea7-82ea-fca136ca4107.png)

**Comment by HelloJocelynLu** [comment]
I would suggest you to take a look at Figure 4 from manuscript. We find that the most effective multi-task training is to combine tasks with same types. It significantly reduces the quality of predicted SMILES, especially when more molecules are generated (i.e.: top-k accuracy when k > 1)

**Comment by MJ-Zeng** [comment]
ok,thanks
